### PR TITLE
Docs: adding info on the accordion markdown container

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Adds info on the accordion markdown container (`::: details`) (I spotted it used [here](https://handsontable.com/docs/modules/#tree-shaking))

[skip changelog]

### CODE REVIEW

File: docs/README-EDITING.md [hunk 1]
```diff
@@ -147,11 +147,12 @@ To render content in different ways, the documentation uses custom Markdown cont
 
 We use the following Markdown containers:
 
-| Container                    | Usage                                                      |
-|------------------------------|------------------------------------------------------------|
-| `::: tip`                    | A note.                                                    |
-| `::: example [options]`      | Renders a code example as specified in [options].          |
-| `::: source-code-link <URL>` | Adds a source code link to the API ref header. |
+| Container                    | Usage                                             |
```
Comment (Siemienik) [inline docs/README-EDITING.md:150]: @kirszenbaum 
I see that you add info about the `details` MD extension. 
According to the Vuepress documentation, there are some additional containers by design:
https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers
and with a custom title: https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers

Want you also describe those here?

Comment (kirszenbaum) [reply docs/README-EDITING.md:150]: @Siemienik - I think we should describe only those containers that we **want** to use (i.e. if a container is not listed in README-EDITING.md, don't use it), not all the containers that we **can** use. That said, there may some more containers that could indeed be useful - I'll have a look at them after the HoT 11.0.0 release

Comment (Siemienik) [reply docs/README-EDITING.md:150]: Ok, please re-request for a review then. I'll keep my fingers crossed. 🤞

Comment (kirszenbaum) [reply docs/README-EDITING.md:164]: @Siemienik - you're right, let's add the remaining containers as well: 66d9e2e

**Review by Siemienik — APPROVED** [review]
Thank you @kirszenbaum, it's all done great 👍 

EDIT: before merging, it is required to add skip changelog into the PR description.

**Pull request merged**